test/transcoding/subgroup_avc_intel_generic.cl

```diff
@@ -0,0 +1,261 @@
+// RUN: %clang_cc1 -O1 -triple spir-unknown-unknown -cl-std=CL2.0 %s -finclude-default-header -emit-llvm-bc -o %t.bc
+// RUN: llvm-spirv %t.bc --spirv-ext=+SPV_INTEL_device_side_avc_motion_estimation -spirv-text -o %t.txt
+// RUN: FileCheck < %t.txt %s --check-prefix=CHECK-SPIRV
+// RUN: llvm-spirv %t.bc --spirv-ext=+SPV_INTEL_device_side_avc_motion_estimation -o %t.spv
+// There is no validation for SPV_INTEL_device_side_avc_motion_estimation implemented in
+// SPIRV-Tools. TODO: spirv-val %t.spv
+// RUN: llvm-spirv -r %t.spv --spirv-ext=+SPV_INTEL_device_side_avc_motion_estimation -o %t.rev.bc
+// RUN: llvm-dis < %t.rev.bc | FileCheck %s --check-prefix=CHECK-LLVM
```

Comment:
I have the same recommendations as in #406:

- generate text representation from `.spv` file, not from `.bc` using `-to-text` option - his way you can omit `--spirv-ext` option and simplify `RUN` line
- no need to pass `--spirv-ext` option to backward translation, all extensions are accepted by default
- try to avoid adding `-finclude-default-header` option if possible - it slows down the test

Reply:
First two points done. Regarding to the third one, it is not possible to properly compile those tests without this option.